contrib/README.md

```diff
@@ -11,6 +11,8 @@ This package includes 4 extensions:
     - Supports the various datatypes in MSSQL.
 - babelfishpg_money
     - supports the `money` type in MSSQL. This is a variation of the opensource fixeddecimal extension.
+-- babelfishpg_unit (Optional)
```

Comment:
This is not really an optional extension i.e. babelfish source code will always be shipped with babelfishpg_unit extension so probably we can add optional part in the build extensions part.

Reply:
Done!